lightningd/txfilter.c

```diff
@@ -52,3 +89,40 @@ bool txfilter_match(const struct txfilter *filter, const struct bitcoin_tx *tx)
 	}
 	return false;
 }
+
+void outpointfilter_add(struct outpointfilter *of, const struct bitcoin_txid *txid, const u32 outnum)
+{
+	struct outpointfilter_entry *op;
+	if (outpointfilter_matches(of, txid, outnum))
+		return;
+	/* Have to mark the entries as notleak since they'll not be
+	 * pointed to by anything other than the htable */
+	op = notleak(tal(of->set, struct outpointfilter_entry));
```

Comment:
OK for now, but better is to add outpointfilter_mark_pointers_used; see chaintopology_mark_pointers_used().